spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -3,7 +3,6 @@
 describe "View authentication requests for a username" do
```

Comment:
I worried that this PR only covers the validation and empty result set, but then I realised George is writing some more tests around this - that's how you decided to split it up?